style.css

```diff
@@ -708,53 +691,56 @@ textarea {
     line-height: 24px;
     letter-spacing: 0;
     color: rgba(52, 69, 99, 1);
-  }
-
-  .deskpar3 {
-    width: 40%;
     margin-left: 6%;
+    width: 58%;
   }
 
-  .languages-1 {
-    margin-left: 12rem;
-    margin-top: -15.5rem;
+  .languages {
+    margin-left: 8.8%;
+    margin-top: 1rem;
   }
 
   .languages-2 {
-    margin-left: -36rem;
-    margin-top: -16rem;
+    margin-right: 4rem;
   }
 
-  .languages-3 {
-    margin-left: 20rem;
-    margin-top: -13rem;
+  .languages-2 li:nth-child(1) {
+    width: 40px;
   }
 
-  .project-1 {
-    margin-left: 5rem;
-    margin-top: -10rem;
+  .languages-2 li:nth-child(2) {
+    width: 100px;
   }
 
-  .project-2 {
-    margin-right: 71%;
-    margin-top: -11rem;
+  .languages-2 li:nth-child(1),
+  .languages-2 li:nth-child(2),
+  .languages-2 li:nth-child(3) {
+    margin-right: -0.5rem;
+    padding: 2%;
   }
 
+  .ruby {
+    display: flex;
+  }
+
+  .project-1,
   .project-3 {
-    margin-left: 5rem;
+    margin-left: 7.8%;
+    margin-top: 3rem;
```

Comment:
- [ ] As mentioned by the previous reviewer, the spacing between your button and the top elements is huge please reduce it. possible way to resolve this is to remove `margin-top: 3rem;` and adjust the top elements accordingly. :slightly_smiling_face: 

The example below shows the difference between yours and the accepted one: :arrow_down: 

 <table>
    <tr>
      <td>Yours </td>
      <td>Accepted type</td>
    </tr>
    <tr>
      <td>

 
![Student cArd](https://github.com/denigit5/Portfolio/assets/43528594/51fc4100-98a1-42e4-9f08-646fae75f5cd)

 
</td>
      <td>

![Figma Card](https://github.com/denigit5/Portfolio/assets/43528594/b55295cf-d724-4464-a981-ed5b2e4016a9)

</td>
    </tr>
  </table>